examples/ignition/etcd.yaml

```diff
@@ -15,12 +15,8 @@ systemd:
             Environment="ETCD_INITIAL_CLUSTER={{.etcd_initial_cluster}}"
             Environment="ETCD_STRICT_RECONFIG_CHECK=true"
 
-{{ if index . "ssh_authorized_keys" }}
-passwd:
-  users:
-    - name: core
-      ssh_authorized_keys:
-        {{ range $element := .ssh_authorized_keys }}
-        - {{$element}}
-        {{end}}
-{{end}}
+
+ignition:
+  config:
+    append:
+      - source: http://172.15.0.2:8080/ignition?profile=authkeys
```

Comment:
Move `http:172.15.0.21:8080/ignition` to metadata variable to avoid hardcoding things in Ignition (Fuze).